### ISSUE TITLE
Unable to resolve module `AccessibilityInfo

### ISSUE THREAD

**Issue by KrifaYounes** [Segware/react-native-mjpeg-player#14]
Unable to resolve module `AccessibilityInfo` from `node_modules/react-native-mjpeg-player/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.

Expo app ejected 

PACKAGE.JSON
    "prop-types": "^15.7.2",
    "react": "~16.9.0",
    "react-dom": "~16.9.0",
    "react-native": "~0.61.5",

    "@babel/core": "^7.9.6",
    "@babel/runtime": "^7.9.6",
    "babel-jest": "~25.2.6",
    "babel-preset-expo": "~8.1.0",

command :
react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res

**Comment by BILLXZY1215** [comment]
![Uploading 截屏2021-08-18 下午2.20.33.png…]()

I have the same problem too...
